### ISSUE TITLE
Interpretation of the success criteria - Where to document?

### ISSUE THREAD

**Issue by DagfinnRomen** [act-rules/act-rules.github.io#297]
We need somewhere to document the interpretation of the success criteria. Now, the interpretation of WCAG is to some extent implicit in the test rules. There needs to be further documentation at the success criteria level.

For each success criterion the following interpretation, we suggest that the following should be documented:
- Scope of the test rules for every success criterion.
- Any assumptions and limitations.
- Whether there are several ways of conforming to the success criterion.
- Parts of a success criterion that is not covered by test rules.

The purpose is to have a clear and concise understanding of what each success criterion entails, and how much is covered by the test rules.

This could be described in a separate document. In lack of a better alternative, we have used the applicability section of the test rules for now.

Also see #287.

**Comment by nitedog** [comment]
The next draft of the ACT Rules Format spec will include a description of the relationship between rules and their accessibility requirements. For example, if the rule fully covers or only partially covers the requirement. With this, it should be possible to construct composite rules that address an entire success criterion. There are already sections to describe assumptions and limitations, which should cover what you outline above.

**Comment by DagfinnRomen** [comment]
Sounds good! Looking forward to the updated ACT rules format.

**Comment by WilcoFiers** [comment]
@DagfinnRomen I think this issue has been resolved with the addition of the Accessibility Requirements Mapping. i'm closing this issue. Feel free to raise a new issue if you have further questions or concerns.
